CONTRIBUTING.md

```diff
@@ -6,3 +6,74 @@ repository before making a change.
 
 Please note we have a [Code of Conduct](CODE_OF_CONDUCT.md), please follow it
 in all your interactions with the project.
+
+
+# Contributing to the blog
```

Comment:
Try to write more general guidelines 🙏 This instructions are the same either it's a blog post contributing our any other kind. Only the section how a blog post file is structured is specific for the blog part.

To make this much more simple we could even have a script that creates a new blog post, just by using a template as a starting point.